### PR TITLE
add arch linux rootless hints to install.md

### PR DESCRIPTION
**Pull request by karfau**
- related to issue #3030
- including pointer to new section in README.md provided by #3038

### CODE REVIEW

**Comment by rhatdan** [comment]
/ok-to-test

@karfau You must sign your PR.

git commit -a -s --amend
git push --force

**Comment by rhatdan** [comment]
Thanks @karfau patch LGTM
@TomSweeneyRedHat @vrothberg @mheon @baude PTAL

**Comment by karfau** [comment]
@rhatdan thx for the notice, I haven't done that before but was under the assumption it would happen when using the github browser UI to change the file.
Will look into it within this week.

The failing test must be some CI flackyness, right?

**Comment by rhatdan** [comment]
@karfau Yes the failures are flakes and we will need to update your PR once they are fixed.  The problem is with changes in some of the tools doing the testing.  There was an upgrade of one, and now it is complaining about something that used to work.

File: install.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 sudo pacman -S podman
 ```
 
+If you have problems when running podman in [rootless](rootless.md) mode follow [these instructions](https://wiki.archlinux.org/index.php/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
```
Comment (TomSweeneyRedHat) [inline install.md]: I'd not include the link to rootless.md.

Comment (karfau) [reply install.md]: Just to make sure I understand your comment: you want me to remove that link, or is this just an expression of opinion that I'm free to (not) consider?

Comment (TomSweeneyRedHat) [reply install.md]: I would like it to be removed unless you or others have strong differing opinions.  That page doesn't tell you how to setup rootless or what it's about, but rather shows you the work that is still outstanding for it.

**Comment by mheon** [comment]
CI is fixed upstream - if you can rebase and fix the signoff issue and the comment from @TomSweeneyRedHat I'd say it's good to merge

File: install.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply install.md]: If you can wait, I'd be hapy if you changed the rootless link to the rootles section being added to the README.md in #3038

**Comment by karfau** [comment]
All ready, but CI still flacky?
Of course if you are checking links inside markdown, my PR would currently contain an invalid one. Nut from trying to understand the logs in the checks I couldn't find out what was the problem.

Thx for the support and let me know whenever there is something I need to do.

**Comment by mheon** [comment]
```
 * 714a2514 "install.md contains hints for rootless setup on arch linux" ... FAIL
  - PASS - commit does not have any whitespace errors
  - FAIL - does not have a valid DCO
```
It looks like one of your commits is missing a signoff - maybe squash them down with `git rebase`?

**Comment by mheon** [comment]
Ah, there's only one commit - `git commit --amend -s` and re-push then

**Comment by karfau** [comment]
Thx, I was very sure I did that, but maybe I missed something, will check

**Comment by TomSweeneyRedHat** [comment]
LGTM and all happy green test buttons.  I think this one is ready to ship.
Thanks @karfau !

**Pull request merged**